tensorflow/contrib/distribute/python/examples/simple_estimator_example.py

```diff
@@ -78,9 +78,8 @@ def predict_input_fn():
     predict_features = tf.data.Dataset.from_tensors([[1.]]).repeat(10)
     return predict_features
 
-  predictions = estimator.predict(input_fn=predict_input_fn)
-  # TODO(anjalsridhar): This returns a generator object, figure out how to get
-  # meaningful results here.
+  prediction_iterable = estimator.predict(input_fn=predict_input_fn)
+  predictions = [prediction_iterable.next() for _ in range(10)]
```

Comment:
Thanks for adding this! Can we add a comment that predictions is a list of output dicts with key as 'logits' and value as an array of model outputs?